Cloud native security overview updates (#3389)
elastic/security-docs: docs/cloud-native-security/cloud-native-security-index.asciidoc

[discrete]
== Container Workload Protection
Provides cloud-native runtime protections for containerized environments by identifying and (optionally) blocking unexpected system behavior in Kubernetes containers. These capabilities are sometimes referred to as container drift detection and prevention. The solution also captures detailed process and file telemetry from monitored containers, allowing you to set up custom alerts and protection rules.

<<d4c-overview, Read the Container workload protection docs>>.

[discrete]
== Cloud Workload Protection for VMs

> Helps you monitor and protect your Linux VMs. It uses {elastic-defend} to instantly detect and prevent malicious behavior and malware, and captures workload telemetry data for process, file, and network activity. You can use this data — along with Elastic's out-of-the-box detection rules and machine learning models — to automate threat identification for your cloud VMs. Alerts that are generated from these detections help reduce the time to identify and remediate threats.

Author: ```suggestion
Helps you monitor and protect your Linux VMs. It uses {elastic-defend} to instantly detect and prevent malicious behavior and malware, and captures workload telemetry data for process, file, and network activity. You can use this data with Elastic's out-of-the-box detection rules and machine learning models. Alerts that are generated from these detections help reduce the time to identify and remediate threats.
```

Reviewer: Just have two more minor suggestions. One is an attribute recommendation and the other is a slight revision to make the last sentence more active:

```suggestion
Helps you monitor and protect your Linux VMs. It uses {elastic-defend} to instantly detect and prevent malicious behavior and malware, and captures workload telemetry data for process, file, and network activity. You can use this data with Elastic's out-of-the-box detection rules and {ml} models. These detections generate alerts that quickly help you identify and remediate threats.
```

Reviewer 2: This suggestion ^ is fine.
